### ISSUE TITLE
Accessibility: View title bar is not accessible with Orca

### ISSUE THREAD

**Issue by Sehun-Kiim** [Issue-Labeling-Bot/Labeling_Bot#162]
Re #91063

OS: Ubuntu 18.04
Screen Reader: Orca
Bug:

Open File Explorer
Tab through the Views, Open Editors, Explorer, Outline, Timeline
Orca doesn't read the name for each view title, while Voice Over on macOS does.

**Comment by Choi-kwang-hyun** [comment]
bug : 0.6654205918312073

**Comment by Choi-kwang-hyun** [comment]
verified : 0.6224693655967712
